ai/main.py

```
@@ -38,21 +55,50 @@ def synopsys(book_id: int, authorization: str = Header(None)):
+        if book_info:
             genre = book_info.get("genre")
             character = book_info.get("character")
             background = book_info.get("background")
-            result = ai.generate_story(genre, character, background, story, choice)
+            result = ai.generate_story(
+                genre = genre,
+                character = character,
+                background = background,
+                choice = request.choice,
+                book_id = book_id)
             return {"status": "success", "result":result}
```

Flake8 (7.2.0)

[error] missing whitespace after ':'

(E231)